include/wx/bookctrl.h

```diff
@@ -38,7 +38,7 @@ enum
     wxBK_HITTEST_NOWHERE = 1,   // not on tab
     wxBK_HITTEST_ONICON  = 2,   // on icon
     wxBK_HITTEST_ONLABEL = 4,   // on label
-    wxBK_HITTEST_ONITEM  = wxBK_HITTEST_ONICON | wxBK_HITTEST_ONLABEL,
+    wxBK_HITTEST_ONITEM  = 16,  // on tab control but not on its icon or label
```

Comment:
These values don't actually have to be bitmasks, do they? I think we should either leave this one as 6 or renumber all of them to go from 0 to 4